### PR TITLE
(feat): New quickstart Temporal cloud

### PR DESCRIPTION
**Pull request by pkudikyala**
**Jira ticket:** https://new-relic.atlassian.net/browse/NR-271064
**Description:** Created new quickstart Temporal cloud.

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/temporal-cloud/FailedWorkflows.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Name of the alert
+name: Failed Workflows
+
+# Description and details
+description: |+
+  This alert is triggered if the Temporal cloud workflows failes once within a 5 minutes window.
```
Comment (sjyothi54) [inline alert-policies/temporal-cloud/FailedWorkflows.yml:6]:
```suggestion
  This alert is triggered if the Temporal cloud workflows failed once within a 5 minutes window.
```

File: quickstarts/temporal-cloud/config.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  Monitoring allows you to identify performance bottlenecks, such as slow workflows or inefficient resource usage, optimize your Temporal cloud for better performance, and help you detect and diagnose faults or errors in your Temporal workflows.
+
+  ## Comprehensive monitoring quickstart for Temporal cloud
+  Temporal cloud provides users with workflow execution states and metadata for debugging purposes. Integrating Temporal cloud with New Relic enhances system health maintenance, early issue identification, and smooth operation assurance. This integration leverages the strengths of both systems to offer a comprehensive monitoring solution, encompassing metrics and alerts. Customize the following steps based on the specific programming language and framework of your Temporal cloud services.
+
+  ## What’s included in this quickstart?
+  New Relic Temporal cloud monitoring quickstart ability to cover quality on out-of-the-box reporting.
+
+  - Dashboards (failed, cancel and terminated workflows, service latency and frontend service requests)
+  - Alerts (CPU utilization, failed workflows and service latency)
```
Comment (sjyothi54) [inline quickstarts/temporal-cloud/config.yml:13]:
```suggestion
  - Alerts (failed workflows and service latency)
```

**Review by sjyothi54 — CHANGES_REQUESTED** [review]
@pkudikyala remove cpu usage alert and make the necessary changes

File: alert-policies/temporal-cloud/FailedWorkflows.yml (hunk 1 above)
Comment (pkudikyala) [reply alert-policies/temporal-cloud/FailedWorkflows.yml]: Hi @sjyothi54 ,
I have updated the review changes.

File: quickstarts/temporal-cloud/config.yml (hunk 2 above)
Comment (pkudikyala) [reply quickstarts/temporal-cloud/config.yml]: Hi @sjyothi54 ,
I have updated the review changes.

**Review by sjyothi54 — DISMISSED** [review]
LGTM

**Review by RamanaReddy8801 — DISMISSED** [review]
LGTM

File: alert-policies/temporal-cloud/FailedWorkflows.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Name of the alert
+name: Failed Workflows
+
+# Description and details
+description: |+
+  This alert is triggered if the Temporal cloud workflows failed once within a 5 minutes window.
```
Comment (seemantk) [inline alert-policies/temporal-cloud/FailedWorkflows.yml:6]:
```suggestion
  This alert is triggered if the Temporal cloud workflows fail once within a 5-minute window.
```

**Review by seemantk — CHANGES_REQUESTED** [review]
minor language adjustment

File: alert-policies/temporal-cloud/FailedWorkflows.yml (hunk 3 above)
Comment (pkudikyala) [reply alert-policies/temporal-cloud/FailedWorkflows.yml]: I updated the request changes.

**Review by seemantk — APPROVED** [review]
Thank you @pkudikyala

**Pull request merged**